runtime/runtime_test.go

```diff
@@ -241,6 +243,22 @@ func (i *testRuntimeInterface) SetProgram(location Location, program *interprete
 	return i.setProgram(location, program)
 }
 
+func (i *testRuntimeInterface) SetInterpreterSharedState(state *interpreter.SharedState) {
+	if i.setInterpreterSharedState == nil {
+		return
+	}
+
+	i.setInterpreterSharedState(state)
+}
+
+func (i *testRuntimeInterface) GetInterpreterSharedState() *interpreter.SharedState {
+	if i.getInterpreterShareState == nil {
```

Comment:
```suggestion
	if i.getInterpreterSharedState == nil {
```

Reply:
Fixed in 3d44ae4a28b25764b4d9b146e25db6f8d280f6a4